setup.py

```diff
@@ -14,7 +14,7 @@
 
 setup(
     name='pyquadkey2',
-    version='0.2.2',
+    version='0.3.2',
```

Comment:
Not sure what the preference is for version bumps here -- I felt like in case there was an issue, I didn't want to increment the bug fix version.